### PR TITLE
Add Advanced Autopilot to Enterprise docs section

### PR DESCRIPTION
**Pull request by rcgenova**

### CODE REVIEW

File: website/source/docs/enterprise/autopilot/index.html.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+# Nomad Enterprise Advanced Autopilot
+
+Nomad Enterprise supports Advanced Autopilot capabilities which enable fully
+automated server upgrades, higher throughput for reads and scheduling, and hot
+server failover on a per availability zone basis. See the sections below for 
+additional details on each of these capabilities. 
+
+* **Automated Upgrades:** Advanced Autopilot enables an upgrade pattern that 
+allows operators to deploy a complete cluster of new servers and then just wait
```
Comment (chelseakomlo) [inline website/source/docs/enterprise/autopilot/index.html.md]: "and then simply wait for the server to complete"

Comment (chelseakomlo) [reply website/source/docs/enterprise/autopilot/index.html.md]: Perhaps write a quick summary (or link to) the open source requirement for upgrades.

File: website/source/docs/enterprise/index.html.md [hunk 2]
```diff
@@ -3,20 +3,24 @@ layout: "docs"
 page_title: "Nomad Enterprise"
 sidebar_current: "docs-enterprise"
 description: |-
-  Nomad Enterprise adds collaboration and governance capabilities to Nomad.
-  Features include Namespaces, Resource Quotas, and Sentinel Policies.
+  Nomad Enterprise adds operations, collaboration, and governance capabilities to Nomad.
```
Comment (dadgar) [inline website/source/docs/enterprise/index.html.md]: operations -> operational?

File: website/source/docs/enterprise/index.html.md [hunk 3]
```diff
@@ -3,20 +3,24 @@ layout: "docs"
...
-multiple teams to safely use a single multi-region deployment, limit resource
-consumption across teams or projects, and enforce arbitrary fine-grained policies
-upon job submission. Nomad Enterprise features include:
+[Nomad Enterprise](https://www.hashicorp.com/products/nomad/) adds operations, 
+collaboration, and governance capabilities to Nomad. Namespaces allow multiple 
+teams to safely use a single multi-region deployment. With Resource Quotas, 
+operators can limit resource consumption across teams or projects. Sentinel 
+policies enable enforcement of arbitrary fine-grained policies on job submission. 
+Advanced Autopilot capabilities enable automated server upgrades, enhanced read 
+scalability, and hot server failover on a per availability zone basis. See the
```
Comment (dadgar) [inline website/source/docs/enterprise/index.html.md]: enhanced read and scheduler scalability

File: website/source/docs/enterprise/index.html.md [hunk 4]
```diff
@@ -3,20 +3,24 @@ layout: "docs"
...
 # Nomad Enterprise
 
-[Nomad Enterprise](https://www.hashicorp.com/products/nomad/) adds collaboration
-and governance capabilities to Nomad. With Nomad Enterprise, operators can enable
-multiple teams to safely use a single multi-region deployment, limit resource
-consumption across teams or projects, and enforce arbitrary fine-grained policies
-upon job submission. Nomad Enterprise features include:
+[Nomad Enterprise](https://www.hashicorp.com/products/nomad/) adds operations, 
+collaboration, and governance capabilities to Nomad. Namespaces allow multiple 
+teams to safely use a single multi-region deployment. With Resource Quotas,
```
Comment (dadgar) [inline website/source/docs/enterprise/index.html.md]: safely use a shared Nomad cluster by giving each team an isolated view of the cluster.

**Pull request merged**